tools/automation/twiml-generator/generator.py

```diff
@@ -24,12 +26,82 @@ def generate_code_sample_filepath(twiml_filepath, language):
     return twiml_filepath.parent.parent / (twiml_filepath.name[:-3] + version + language_spec['extension'])
 
 
+def test(code_generator):
```

Comment:
@smendes Does that means, that we should remove the code for testing Java/C# here when the twiml-generator includes the proper testing for those languages?